docs/cloud/google-cloud.rst

```diff
@@ -0,0 +1,84 @@
+Running Mango on Google Cloud
+=============================
+
```

Comment:
how do you make a google cloud cluster?

Reply:
refer to the section below, there's a step to 
`Create the Cloud Dataproc Cluster`